packages/pages/src/dev/server/middleware/indexPage.ts

```diff
@@ -263,8 +263,8 @@ const createEntityPageListItems = (
         </td>
         <td>
           ${
-            accountId
-              ? `<a href="${formatContentLink(uid)}"> ${entityId} </a>`
+            accountId && universe
+              ? `<a href=${formatContentLink(uid)}> ${entityId} </a>`
```

Comment:
You still need quotes around the href I believe. Remove the spaces around ${entityId}

Reply:
Done